### PR TITLE
Add and clean up tests

### PR DESCRIPTION
**Pull request by thpr**

### CODE REVIEW

File: PCGen-base/code/src/test/pcgen/base/format/ArrayFormatManagerTest.java [hunk 1]
```diff
@@ -218,4 +220,54 @@ public void testGetComponent()
 	{
 		assertEquals(new NumberManager(), manager.getComponentManager());
 	}
+
+	public void testIsDirect()
```
Comment (karianna) [inline PCGen-base/code/src/test/pcgen/base/format/ArrayFormatManagerTest.java]: What aspect of Directness is it testing?  Is it testArrayFormatManagerIsaDAG?

File: PCGen-base/code/src/test/pcgen/base/formatmanager/FormatUtilitiesTest.java [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+public class FormatUtilitiesTest extends TestCase
+{
+
+	public void testConstructor()
+	{
+		TestSupport.invokePrivateConstructor(FormatUtilities.class);
+	}
+
+	@Test
+	public void testIsValid()
```
Comment (karianna) [inline PCGen-base/code/src/test/pcgen/base/formatmanager/FormatUtilitiesTest.java]: Is what valid?  A FormatManager?

File: PCGen-base/code/src/test/pcgen/base/formatmanager/FormatUtilitiesTest.java [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+			FormatUtilities.isValid(new BadManager2());
+			fail("Should reject a manager without a managed class");
+		}
+		catch (NullPointerException e)
+		{
+			//expected
+		}
+	}
+
+	private class BadManager1 implements FormatManager<String>
```
Comment (karianna) [inline PCGen-base/code/src/test/pcgen/base/formatmanager/FormatUtilitiesTest.java]: Perhaps describe in the class name what's bad about it.  And this is a FormatManager right?

File: PCGen-base/code/src/test/pcgen/base/formatmanager/FormatUtilitiesTest.java [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+
+		@Override
+		public FormatManager<?> getComponentManager()
+		{
+			return null;
+		}
+
+	}
+
+	private class BadManager2 implements FormatManager<String>
```
Comment (karianna) [inline PCGen-base/code/src/test/pcgen/base/formatmanager/FormatUtilitiesTest.java]: Same as BadManager1

File: PCGen-base/code/src/test/pcgen/base/graph/util/EdgeUtilitiesTest.java [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+	@Override
+	protected void setUp() throws Exception
+	{
+		node1 = new Integer(3);
+		node2 = new Integer(4);
+		node3 = new Integer(6);
+		edge1 = new DefaultHyperEdge<>(Arrays.asList(new Integer[]{node2, node1, node3}));
+	}
+
+	public void testGetNodeAt()
```
Comment (karianna) [inline PCGen-base/code/src/test/pcgen/base/graph/util/EdgeUtilitiesTest.java]: Test name could reflect what it does, e.g. testNodesPlacedInGraphAreOnCorrectEdges or something like that.

**Comment by thpr** [comment]
Some name changes made - the rest of these are test<methodname>.  This is rather consistent today throughout the entire test system for unit tests, so if we want to change that then it should be done as a bulk change to keep things cross-code-base consistent

Closing due to conflicts, Will open a new PR.

**Pull request closed without merging**